rubocop-shopify.gemspec

```diff
@@ -19,7 +19,8 @@ Gem::Specification.new do |s|
   s.metadata = {
     "source_code_uri" => "https://github.com/Shopify/ruby-style-guide/tree/v#{s.version}",
     "allowed_push_host" => "https://rubygems.org",
+    "rubygems_mfa_required" => "true",
```

Comment:
I think we can't enable this for our Shopify gems since `shipit-shopify` can't use MFA yet.